crates/ruma-client-api/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # [unreleased]
 
+
+Breaking changes:
+
+*  `DeviceList` has moved from `sync::sync_events::v3` to `sync::sync_events`
```

Comment:
With the previous review comment applied, this can become something like:

```suggestion
Improvements:

* `DeviceList` has moved from `sync::sync_events::v3` to `sync::sync_events`
  * It is still available under the old location for backwards compatibility
```